easybuild/easyconfigs/s/SQLite/SQLite-3.10.0-foss-2015a.eb

```diff
@@ -0,0 +1,40 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'SQLite'
+version = '3.10.0'
+
+homepage = 'http://www.sqlite.org/'
+description = 'SQLite: SQL Database Engine in a C Library'
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+# eg. http://www.sqlite.org/2014/sqlite-autoconf-3080600.tar.gz
+source_urls = ['http://www.sqlite.org/2016/']
+version_minor_etc = version.split('.')[1:]
+version_minor_etc += '0' * (3 - len(version_minor_etc))
+version_str = '%(version_major)s' + ''.join('%02d' % int(x) for x in version_minor_etc)
+sources = ['sqlite-autoconf-%s.tar.gz' % version_str]
+
+dependencies = [
+    ('libreadline', '6.3'),
+    ('Tcl', '8.6.4'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/sqlite3'],
```

Comment:
@valtandor: please merge https://github.com/valtandor/easybuild-easyconfigs/pull/6 to extend sanity check (like in existing SQLite easyconfigs)